pennylane/ops/identity.py

```diff
@@ -53,8 +53,8 @@ class Identity(CVObservable, Operation):
     def _flatten(self):
         return tuple(), (self.wires, tuple())
 
-    def __init__(self, *params, wires=None, id=None):
```

Comment:
I dont really understand how the initialization like `qml.Identity(0)` worked with this before 🤔

Reply:
https://github.com/PennyLaneAI/pennylane/blob/5cfdeddbb59a7147146a0c550b8e1099de0cc022/pennylane/operation.py#L1053